modules/ROOT/pages/scheduler-concept.adoc

```diff
@@ -5,15 +5,16 @@ endif::[]
 
 The Scheduler component enables you to trigger a flow when a time-based condition is met. You can configure it to be triggered at a regular interval or give it a more flexible cron expression. For example, a Scheduler might trigger an event to start a flow every 5 seconds.
 
-Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
+Consider the following when you implement a Scheduler in your Mule application:
 
+* Schedulers use the same timezone as the machine on which Mule is running. However, if an application is running in CloudHub, the Scheduler conforms to the UTC timezone, regardless of the geographic region in which the application is running.
 // TODO: MULE-14438
-The scheduler component can trigger only one flow at any given time, and if a triggered flow is executing, it won't be triggered again until it finishes. If xref:execution-engine.adoc#backpressure[back-pressure] is triggered
-because no resources are available at the time of the trigger, that trigger
-will be skipped.
-
+* The scheduler component can trigger only one flow at any given time, and if a triggered flow is executing, it won't be triggered again until it finishes processing.
+* If xref:execution-engine.adoc#backpressure[back-pressure] is triggered because no resources are available at the time of the trigger, that trigger is skipped.
 //Enhancement request for this: MULE-14930
-Note that in a Mule runtime cluster or multi-worker CloudHub deployment, the Scheduler runs (or triggers) only on the primary node (that is, only in one Mule runtime).
+* In a Mule runtime engine cluster or multi-worker CloudHub deployment, the Scheduler runs (or triggers) only on the primary node (that is, only in one Mule instance).
```

Comment:
So, "triggers" requires an object noun, so it needs to be something like "...the Scheduler runs (or is triggered) only on..
or "...the Scheduler runs (or triggers something) only on...